### DISCUSSION TITLE
Different committer email and author email used by the bot?

### DISCUSSION THREAD

**Discussion by Moulick** [renovatebot/renovate#24238] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Describe the problem

Currently we see that Renovate creates commits with different author and committer emails. In our repos we see that the commits are created with author `<number>+renovate[bot]@users.noreply.github.com` and committer email as `noreply@github.com`. You can see this by running `git show -s --format="Author: %aE, Committer: %cE"`. 

First question, why? Given that both emails cannot receive an email anyways, why is the committer email `noreply@github.com` instead being the same as Author email? Even more, `noreply@github.com` does not make sense to be the committer email by Renovate Bot anyways ❌

### Have you created a minimal reproduction repository?

Run `git show -s --format="Author: %aE, Committer: %cE"` in a branch created by renovate

**Comment by rarkins** [comment, marked as answer]
It's a side effect of the platformCommit option. Without it, you won't get signed bot commits. GitHub assigns and controls the email addresses

**Reply by Moulick** [reply]
So you mean the comits are signed by a gpg key with email `noreply@github.com` ?

**Reply by rarkins** [reply]
They're signed by GitHub. What you see is what they decide
